From fe605b62d9190e3ece8742a650c54823d8d16d5f Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 8 May 2018 11:07:07 +0800
Subject: [PATCH 3748/5242] MLK-18229-2 gpu: imx: dpu: fetcheco: Cosmetic
 changes on fe_ops entries

commit  a5d632fcbd985f49a4299fc163b00ac24d2d57ff from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch contains cosmetic changes on fe_ops entries to make the
entries be more readable.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-fetcheco.c |   32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-fetcheco.c b/drivers/gpu/imx/dpu/dpu-fetcheco.c
index 3379909..903fddc 100644
--- a/drivers/gpu/imx/dpu/dpu-fetcheco.c
+++ b/drivers/gpu/imx/dpu/dpu-fetcheco.c
@@ -351,22 +351,22 @@ void dpu_fe_put(struct dpu_fetchunit *fu)
 EXPORT_SYMBOL_GPL(dpu_fe_put);
 
 static const struct dpu_fetchunit_ops fe_ops = {
-	.set_burstlength = fetchunit_set_burstlength,
-	.set_baseaddress = fetchunit_set_baseaddress,
-	.set_src_bpp = fetchunit_set_src_bpp,
-	.set_src_stride = fetchunit_set_src_stride,
-	.set_src_buf_dimensions = fetcheco_set_src_buf_dimensions,
-	.set_fmt = fetcheco_set_fmt,
-	.enable_src_buf = fetchunit_enable_src_buf,
-	.disable_src_buf = fetchunit_disable_src_buf,
-	.is_enabled = fetchunit_is_enabled,
-	.set_framedimensions = fetcheco_set_framedimensions,
-	.set_controltrigger = fetcheco_set_controltrigger,
-	.get_stream_id = fetchunit_get_stream_id,
-	.set_stream_id = fetchunit_set_stream_id,
-	.pin_off = fetchunit_pin_off,
-	.unpin_off = fetchunit_unpin_off,
-	.is_pinned_off = fetchunit_is_pinned_off,
+	.set_burstlength	= fetchunit_set_burstlength,
+	.set_baseaddress	= fetchunit_set_baseaddress,
+	.set_src_bpp		= fetchunit_set_src_bpp,
+	.set_src_stride		= fetchunit_set_src_stride,
+	.set_src_buf_dimensions	= fetcheco_set_src_buf_dimensions,
+	.set_fmt		= fetcheco_set_fmt,
+	.enable_src_buf		= fetchunit_enable_src_buf,
+	.disable_src_buf	= fetchunit_disable_src_buf,
+	.is_enabled		= fetchunit_is_enabled,
+	.set_framedimensions	= fetcheco_set_framedimensions,
+	.set_controltrigger	= fetcheco_set_controltrigger,
+	.get_stream_id		= fetchunit_get_stream_id,
+	.set_stream_id		= fetchunit_set_stream_id,
+	.pin_off		= fetchunit_pin_off,
+	.unpin_off		= fetchunit_unpin_off,
+	.is_pinned_off		= fetchunit_is_pinned_off,
 };
 
 void _dpu_fe_init(struct dpu_soc *dpu, unsigned int id)
-- 
1.7.9.5

